### ISSUE TITLE
Feature Request: Hide dir if it not allow in user rules

### ISSUE THREAD

**Issue by xiaolei0125** [hacdias/webdav#67]
Can hide some dir(not show in webdav client) if these dir was not allow in user rules config? 
  I don't wants some users know the dir exist if they has no permisson for those dir.  Thanks

**Comment by changfeng98** [comment]
Have you found a solution to this problem? I want to to solve the problem too.

**Comment by Pranoypauljr** [comment]
@hacdias isnt this feature implemented recently by you using permissions_fs.go and permissions.go, i was planning to implement it then say your commit in the main branch:), its time to close this right?

**Comment by hacdias** [comment]
Yes, it was actually a side-effect from closing a few security issues 😅
